src/content/docs/iast/use-iast.mdx

```diff
@@ -0,0 +1,189 @@
+---
+title: New Relic interactive applications security testing (IAST)
+tags:
+  - IAST
+  - Exploitable vulnerabilities
+metaDescription: Use Vulnerability Management to find exploitable vulnerabilities in your application.
+---
+
+import iastSummary from 'images/iast_screenshot-crop_summary.webp'
+
+import iastInstall from 'images/IAST_screenshot-crop_install.webp'
+
+import iastAppVuln from 'images/IAST_screenshot-crop_app-vuln.webp'
+
+import iastAppCoverage from 'images/IAST_screenshot-crop_app-coverage.webp'
+
+import iastVulnDetails from 'images/iast_screenshot-crop_vuln-details.webp'
+
+import iastExploitablesAll from 'images/iast_screenshot-crop_exploitables-all.webp'
+
+import iastCoverageOverview from 'images/iast_screenshot-crop_coverage-overview.webp'
+
+import iastUntestedApps from 'images/iast_screenshot-crop_untested-apps.webp'
+
+import iastEnableIast from 'images/iast_screenshot-crop_enable-iast.webp' 
+
+<Callout title="PREVIEW"> 
```

Comment:
Is this still in preview?

Reply:
Do we want to provide some info on what someone can do to sign up to try it out?

Reply:
Yes, it'll be a public preview. Users don't need to do anything, it'll be in the UI when they release it.